chat-plugins/hangman.js

```diff
@@ -303,8 +339,10 @@ exports.commands = {
 	guess: function (target, room, user) {
 		if (!room.game) return this.errorReply("There is no game running in this room.");
 		if (!this.canTalk()) return;
+		// @ts-ignore
 		if (!room.game.guess) return this.errorReply("You can't guess anything in this game.");
 
+		// @ts-ignore
 		room.game.guess(target, user);
```

Comment:
If this is a generic command, it should go next to wherever `/choose` is defined.